osfoffline/polling_osf_manager/osf_query.py

```diff
@@ -17,10 +17,11 @@
 
 
 class OSFQuery(object):
-    def __init__(self, loop, oauth_token):
+    def __init__(self, loop, oauth_token, limit=5):
         self.headers = {
             'Authorization': 'Bearer {}'.format(oauth_token),
         }
+        self.throttler = asyncio.Semaphore(limit)
```

Comment:
> No reason for it to be. The additional functionality provided BoundedSemaphore by is pointless here, we only care about upper limits.

Hmm. From the description, I assumed that upper (not just lower) limits were precisely what `BoundedSemaphore` would provide. Will look into that when I'm less tired then. :)